pipelines/zendesk/helpers/api_helpers.py

```diff
@@ -92,14 +91,14 @@ def basic_load(resource_api: Iterator[Any]) -> Iterator[TDataItem]:
     """
     # sometimes there is a single element which is a dict instead of a generator of objects
     if isinstance(resource_api, dict):
-        yield resource_api
+        yield _make_json_serializable(resource_api)
```

Comment:
every Zenpy object derives from `BaseObject` that implements this:
```
    def to_json(self, indent=2):
        """ Return self formatted as JSON. """
        return json.dumps(self, default=json_encode_for_printing, indent=indent)
        ```
why not to use it? `Ticket` does that as well

Reply:
Hey @rudolfix,

From what I can see from experimenting with it, the to_json method will convert the whole object to a string which would then need to be converted to a dict with the correct data types for all fields. This is a bit redundant because the process is done by the to_dict method. In my opinion, using to_json and then encoding ourselves does more harm then good.

Reply:
I had a look at that implementation for to_dict. I think using to_dict with the default options should be ok for our use case.